infra/stacks/cloud-storage-service/index.ts

```diff
@@ -57,6 +57,10 @@ const syncServiceAuthKeyArn: pulumi.Output<string> = aws.secretsmanager
   .getSecretVersionOutput({ secretId: SYNC_SERVICE_AUTH_KEY })
   .apply((secret) => secret.arn);
 
+const PROPERTIES_SERVICE_AUTH_KEY = config.require(`properties_service_auth_key`);
+// Reuse internalApiKeyArn since properties_service_auth_key points to the same secret
+const propertiesServiceAuthKeyArn = internalApiKeyArn;
```

Comment:
Properties service has the exact same auth key as everything else like dss so you can just use that which already exists here.